docs-source/content/security/secrets.md

```diff
@@ -71,14 +72,36 @@ username:  8 bytes
 #### WebLogic domain image pull secret
 
 The WebLogic domain that the operator manages can have images that are protected
-in the registry. The `imagePullSecrets` setting can be used to specify the
+in the registry. The `imagePullSecrets` setting on the `Domain` can be used to specify the
```

Comment:
by `Domain` - you mean 'on the domain resource' - right?

Reply:
Yes, I clarified this in adding the operator pull secret section, if you follow the link referenced after this you see example and more context